# Engine id to check if exists
string id
---
# Exists == 1, Does not exist == 0
int32 exists
